ui/src/app/pages/profile/profile-page.tsx

```diff
@@ -577,8 +577,8 @@ export const ProfilePage = withUserProfile()(class extends React.Component<
                     <div>Remaining <i>All of Us</i> free credits:</div>
                 </FlexColumn>
                 <FlexColumn style={{alignItems: 'flex-end', marginLeft: '1.0rem'}}>
-                    <div style={{marginTop: '0.4rem'}}>{renderUSD(profile.freeTierUsage)}</div>
```

Comment:
renderUSD() is strictly a string processing function now.  The formatting is desired only here, so it was moved here.